frontend/common/src/liquity/index.ts

```diff
@@ -31,11 +32,32 @@ export type LiquityStakingDetails = z.infer<typeof LiquityStakingDetails>;
 export const LiquityPoolDetail = z.object({
   gains: AssetBalance,
   rewards: AssetBalance,
-  deposited: AssetBalance
+  deposited: AssetBalance,
+  proxyTo: z.string().nullish()
 });
 
 export type LiquityPoolDetail = z.infer<typeof LiquityPoolDetail>;
 
 export const LiquityPoolDetails = z.record(LiquityPoolDetail);
 
 export type LiquityPoolDetails = z.infer<typeof LiquityPoolDetails>;
+
+export const LiquityStatistic = z.object({
+  totalUsdGainsStabilityPool: NumericString,
+  totalUsdGainsStaking: NumericString,
+  totalDepositedStabilityPool: NumericString,
+  totalWithdrawnStabilityPool: NumericString,
+  totalDepositedStabilityPoolUsdValue: NumericString,
+  totalWithdrawnStabilityPoolUsdValue: NumericString,
+  stakingGains: z.array(AssetBalance),
+  stabilityPoolGains: z.array(AssetBalance)
+});
+
+export type LiquityStatistic = z.infer<typeof LiquityStatistic>;
+
+export const LiquityStatistics = z.object({
```

Comment:
What would be an alternative to the name so that we can free the plural for the type above? Can you think fo anything?

Reply:
`LiquityStatisticDetail` & `LiquityStatisticDetails`? maybe?😆